src/angular/components/view.component.ts

```diff
@@ -225,4 +228,14 @@ export class ViewComponent implements OnDestroy, OnInit {
             await this.totpUpdateCode();
         }
     }
+
+    private checkPasswordExposed() {
+        if (this.cipher.type === CipherType.Login && this.cipher.login.password != null) {
```

Comment:
I am not sure that people will want this running automatically. Some people have a problem with sending hashes to HIBP and choose not to click the button.

Reply:
Yea i considered that, and there should probably be an option perhaps default to off, and suggest to the users that they may opt-in to provide some additional features?